grafana-formula/grafana/files/datasources.yml

```diff
@@ -1,17 +1,15 @@
 # config file version
-apiVersion: 1
-
-deleteDatasources:
-  - name: Prometheus
-    orgId: 1
+apiVersion: 2
```

Comment:
Is this change mandatory ? Does version `2` work with Grafana  6.2.5 (the one we support) ?

Reply:
Sorry, my bad. Fixed